src/components/Match/Match.jsx

```diff
@@ -29,60 +37,30 @@ import {
   runesColumns,
 } from './matchColumns.jsx';
 import BuildingMap from '../BuildingMap/BuildingMap';
-import { defaultSort } from '../../utility';
+import {
+  defaultSort,
+} from '../../utility';
 // import { TabBar } from '../TabBar';
 
-const match = (state) => state[REDUCER_KEY].match;
-const getMatchPlayers = (state) => state[REDUCER_KEY].match.match.players;
-const getSortState = (state) => state[REDUCER_KEY].match.sortState;
-const getSortField = (state) => state[REDUCER_KEY].match.sortField;
-const getSortFn = (state) => state[REDUCER_KEY].match.sortFn;
+const match = (state) => state.app.match;
+const getMatchPlayers = (state) => state.app.match.match.players;
+const getSortState = (state) => state.app.match.sortState;
+const getSortField = (state) => state.app.match.sortField;
+const getSortFn = (state) => state.app.match.sortFn;
 const sortMatchPlayers = (state) => defaultSort(getMatchPlayers(state), getSortState(state), getSortField(state), getSortFn(state));
```

Comment:
you might want to move these selectors to the reducer since the match reducer is what cares about the state shape, not the component.